assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -70,10 +70,12 @@
 			height: 34px;
 			margin: 0;
 			padding: 0;
+			padding-right: 32px;
```

Comment:
@hussain-t let's update the existing `padding` value instead of adding an additional `padding-right` property:

```diff
- padding: 0;
+ padding: 0 32px 0 0;